sdk/storage/Azure.Storage.Blobs/AzureStorageNetMigrationV12.md

```diff
@@ -612,6 +613,58 @@ byte[] transactionalMD5 = response.Value.Details.ContentHash;
 // validate stream against hash in your workflow
 ```
 
+### Resiliency
+
+#### Retry policy
+
+V11
+
+```csharp
+Stream targetStream = new MemoryStream();
+BlobRequestOptions options = new BlobRequestOptions()
+{
+    RetryPolicy = new ExponentialRetry(TimeSpan.FromSeconds(10), 6)
```

Comment:
Is this client side or server side?

Reply:
that's not timeout, added explicit names.